src/components/file-upload/file-upload.css

```diff
@@ -0,0 +1,67 @@
+.dropzone-border-left {
+    left: -1px;
+    top: -1px;
+    bottom: -1px;
+    width: 2px;
+    content: "";
+    position: absolute;
+    transform: scaleY(0);
+    transition: transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
+    pointer-events: none;
+    background-color: #6a1b9a;
+}
+
+.dropzone-border-right {
```

Comment:
Could you merge redundant styles? For example `.dropzone-border-left` and `.dropzone-border-right` seem to be the same. Why we started using CSS instead of  a style callback?